src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -38,30 +78,68 @@ import braintreeDepositAndCheckout from '@/graphql/mutation/braintreeDepositAndC
 import KvButton from '@/components/Kv/KvButton';
 import KvIcon from '@/components/Kv/KvIcon';
 import BraintreeDropInInterface from '@/components/Payment/BraintreeDropInInterface';
+import KvCheckbox from '@/components/Kv/KvCheckbox';
+import { validationMixin } from 'vuelidate';
+import { required, email } from 'vuelidate/lib/validators';
 
 export default {
 	components: {
 		KvButton,
 		KvIcon,
-		BraintreeDropInInterface
+		BraintreeDropInInterface,
+		KvCheckbox
 	},
 	inject: ['apollo', 'cookieStore'],
 	mixins: [
-		checkoutUtils
+		checkoutUtils,
+		validationMixin
 	],
 	props: {
 		amount: {
 			type: String,
 			default: ''
+		},
+		isGuestCheckout: {
+			type: Boolean,
+			default: false
 		}
 	},
 	data() {
 		return {
+			email: null,
+			termsAgreement: false,
 			enableCheckoutButton: false,
+			displayEmailErrors: false,
+			displayTermsAgreementErrors: false,
```

Comment:
@emuvente I had tried that before, but the problem is that it was showing the error message as soon as the page loaded. This started to happen when i added the "required" validation. Any suggestions on how to resolve that? Thanks

Reply:
Were you using `$invalid` or `$error`? Because those behave slightly differently. `$error` will be false until the input is interacted with or `$touch()` is called, whereas `$invalid` can be true immediately. That's why I recommend using `$error` for the `v-if`.

Reply:
@emuvente Ok, i was finally able to get it to work

Reply:
I was using $invalid. It works now. thanks 👍